### PR TITLE
Remove left side extra space of toolbar title with navigation

### PR DESCRIPTION
**Pull request by chibatching**
## Issue
- none

## Overview (Required)
- set content inset 0dp to remove extra margin between navigation button and toolbar title
  - I prefer small margin but this doesn't follow material design guideline. If it is unacceptable, please close this PR.

## Links
- none

## Screenshot
Before | After
:--: | :--:
 | 
 | 
 | 
 | 
 | 
 |

### CODE REVIEW

**Comment by konifar** [comment]
> I prefer small margin but this doesn't follow material design guideline.

I know what you mean. Actually I'm for your opinion.

File: app/src/main/res/layout/activity_sponsors.xml [hunk 1]
```diff
@@ -13,6 +13,7 @@
                 style="@style/BaseToolbar"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
+                app:contentInsetStartWithNavigation="0dp"
```
Comment (konifar) [inline app/src/main/res/layout/activity_sponsors.xml]: Good. How about adding this to `@style/BaseToolbar`?

```xml
<item name="contentInsetStartWithNavigation"> 0dp </item>
```

Comment (chibatching) [inline app/src/main/res/layout/activity_sponsors.xml]: Oh, I've missed toolbar style. I agree it!

Comment (chibatching) [inline app/src/main/res/layout/activity_sponsors.xml]: Fix af0b354

**Comment by konifar** [comment]
Perfect! LGTM!
If someone doesn't like this style, they can send another PR.

**Pull request merged**